intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -708,16 +708,18 @@ \subsection{Creating a new VO}
     \includegraphics[width=\textwidth]{ch6-VO-new}
 \end{figure}
 
-\subsection{Request more storage space}
+\subsection{Requesting more storage space}
 
 If you're a moderator of a VO, you can request additional quota for the VO and
 its members.
 
 \begin{enumerate}
     \item Go to \url{https://account.vscentrum.be/django/vo/edit} and scroll down
         to ``Request additional quota''. See the screenshot below to see how this looks.
-    \item Fill out how much \emph{additional} storage you want.
-    \item Add a comment explaing why you need additional storage space and submit
+    \item Fill out how much \emph{additional} storage you want. In the screenshot below,
+        we're asking for 500 GiB extra space for \verb|VSC_DATA|, and for 1 TiB extra
+        space on \verb|VSC_SCRATCH_KYUKON|.
```

Comment:
@redfast00 Open an issue to clarify in the section where we expain `$VSC_SCRATCH` what `kyukon` is